bundles/org.openhab.binding.evcc/src/main/resources/OH-INF/i18n/evcc.properties

```diff
@@ -104,8 +106,3 @@ channel-type.evcc.vehicleSoC.label = Vehicle SoC
 channel-type.evcc.vehicleSoC.description = Current State of Charge of EV
 channel-type.evcc.vehicleTitle.label = Vehicle title
 channel-type.evcc.vehicleTitle.description = Name of EV
-
-# channel types
-
-offline.configuration-error.no-host = No host configured
```

Comment:
Why removed? Seems still referenced here:
https://github.com/openhab/openhab-addons/blob/fe9f0583b5d8dcc88d40c9517f63f327b61261b8/bundles/org.openhab.binding.evcc/src/main/java/org/openhab/binding/evcc/internal/EvccHandler.java#L197-L198

Reply:
I regenerated the file using `mvn i18n:generate-default-translations` and this section was removed then, I obviously didn't notice. I've reverted this, see https://github.com/openhab/openhab-addons/pull/14245/commits/8142ecd45648689ad88bfde47a2687ec83c584b9.

Reply:
I wonder why it did that - does it happen if you run it again? If yes, can you try to fix the header (line 108) to:

```
# thing status descriptions
```

Maybe the tool is using the header, which is currently wrong, so that could explain it?

Reply:
Hmm, that's weird. If I run again, it doesn't change anything. I guess I probably did something else with this file...
Anyway, I'll correct that header.